### ISSUE TITLE
Making NetherSX2 From Scratch ? 

### ISSUE THREAD

**Issue by Ottorock** [Trixarian/NetherSX2-patch#93]
What year are you gonna do that Ima be excited and rooting for you

**Comment by Trixarian** [comment]
Yes. It's just been slow going since my free time has been really limited the last couple of months
